src/Components/CompleteProfile/CompleteProfileCollectionDialog.tsx

```diff
@@ -8,10 +17,39 @@ interface CompleteProfileCollectionDialogProps {
 export const CompleteProfileCollectionDialog: FC<CompleteProfileCollectionDialogProps> = ({
   onClose,
 }) => {
+  const {
+    contextPageOwnerId,
+    contextPageOwnerSlug,
+    contextPageOwnerType,
+  } = useAnalyticsContext()
+  const { trackEvent } = useTracking()
+
+  const handleSuccess = () => {
+    const editedUserProfile: EditedUserProfile = {
+      action: ActionType.editedUserProfile,
+      context_screen: ContextModule.inquiry,
+      context_screen_owner_type: contextPageOwnerType,
+      platform: "web",
+    }
+
+    const tappedMyCollectionAddArtworkArtist: TappedMyCollectionAddArtworkArtist = {
+      action: ActionType.tappedMyCollectionAddArtworkArtist,
+      context_screen: OwnerType.myCollectionAddArtworkArtist, // FIXME: Should probably be ContextModule?
+      context_module: ContextModule.myCollectionAddArtworkAddArtist,
+      context_screen_owner_id: contextPageOwnerId,
+      context_screen_owner_slug: contextPageOwnerSlug,
+      platform: "web",
+    }
+
+    trackEvent(editedUserProfile)
+    trackEvent(tappedMyCollectionAddArtworkArtist)
```

Comment:
Side non-blocking note: it would be great to collect all of these distributed tracking calls within their own hook. (not sure how many there are, but its always easier to manage if things are in one place.)